doc/internal/man3/ossl_algorithm_do_all.pod

```diff
@@ -41,11 +41,11 @@ The function described here are mainly useful for discovery, and
 possibly display of what has been discovered, for example an
 application that wants to display the loaded providers and what they
 may offer, but also for constructors, such as
-L<ossl_construct_method(3)>.
+L<ossl_method_construct(3)>.
 
 =head1 SEE ALSO
 
-L<ossl_construct_method(3)>, L<EVP_MAC_do_all(3)>
+L<ossl_method_construct(3)>, L<EVP_MAC_do_all_ex(3)>
```

Comment:
EVP_MAC_do_all_ex is EVP_MAC_do_all_provided now.

Reply:
Does that mean that a rebase is needed, or did I forget to update this link. Either way, @richsalz, if you wish, feel free to correct it. Otherwise, I will (since it's my omission to start with)

Reply:
things morphed while this PR was dead. :)  I fixed it.  Thanks @t8m.